src/sdk/cpp/lib/src/event.cpp

```diff
@@ -70,7 +74,7 @@ DiscoverMessage DiscoverMessage::FromJson(const json& j) {
         version = j["version"].get<std::string>();
     } else {
         version = "0.0.0";
-        log::Warn() << "Discover Message API doesn't fit the sdk version. Please update to avoid unknown behavior.";
+        logger.Warn() << "Discover Message API doesn't fit the sdk version. Please update to avoid unknown behavior.";
```

Comment:
maybe the consistency of the name of the logger instance could be improved: here it is only logger, in client.cpp it is "client_logger".